tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogUtil.java

```diff
@@ -552,6 +552,16 @@ public static AlterTableDesc setProperty(String tableName, KeyValueSet params, A
     return alterTableDesc;
   }
 
+  public static AlterTableDesc unsetProperty(String tableName, String[] propertyKeys, AlterTableType alterTableType) {
```

Comment:
I know. But I did it for the consistency with the other AlterTable methods: CatalogUtil#renameColumn, renameTable, addNewColumn and setProperty. How about clear this issue with AlterTableNode#clone? These methods worth to be refined.